pyoutline/outline/config.py

```diff
@@ -22,13 +22,17 @@
 
 from future import standard_library
 standard_library.install_aliases()
+import getpass
 import os
 
 from six.moves.configparser import SafeConfigParser
 
 
 __all__ = ["config"]
 
+PYOUTLINE_ROOT_DIR = os.path.dirname(os.path.dirname(__file__))
+DEFAULT_USER_DIR = '/tmp/opencue/outline/user/{}'.format(getpass.getuser())
```

Comment:
Should we use something like [`tempfile`](https://docs.python.org/3/library/tempfile.html) for this, to help with OS compatibility?

Reply:
Good call, thanks!